postgraphile/website/postgraphile/best-practices.md

```diff
@@ -66,7 +66,7 @@ GRANT UPDATE ON users TO graphql_role;
 ```
 
 Column-level SELECT grants
-[cause a lot of issues](./requirements/#dont-use-column-based-select-grants)
+[cause a lot of issues](./requirements#your-postgresql-database)
```

Comment:
Why was this changed? We specifically added an anchor so we could link to that bullet:

```
- **<a name="dont-use-column-based-select-grants"></a>Don't use column-based
  SELECT grants**:
```

Reply:
I thought the anchor had changed, didn't spot it was a link within the page which wasn't a heading!

Reply:
Ah, this is because, even though linking to `./requirements/#dont-use-column-based-select-grants` works, I didn't realise that is what it was doing. Visually, the page scrolls too far such that the first line of the bullet point is hidden behind the navbar. On Firefox, at least. 

![Screenshot 2025-02-05 at 16 57 16](https://github.com/user-attachments/assets/b0bbe2cd-a09c-47a9-91d5-074d05766dc3)

Reply:
![Screenshot 2025-02-05 at 17 01 17](https://github.com/user-attachments/assets/a62f7f61-769c-4d7a-b845-38bf08cdfe62)

On Chrome, it doesn't scroll to anywhere...

Reply:
Weiiiiird. Okay whatever :man_shrugging: